promql/parser/ast.go

```diff
@@ -208,6 +208,12 @@ type VectorSelector struct {
 	UnexpandedSeriesSet storage.SeriesSet
 	Series              []storage.Series
 
+	// SelectHints are the SelectHints to use when selecting corresponding info series, if enabled.
```

Comment:
Describe what it is, not how you expect to use it.

Reply:
Tried to rephrase it, PTAL.

Reply:
Dropped this field.